pr2eus_tutorials/README.md

```diff
@@ -109,6 +109,15 @@ It means the robot now recognizes each objects as individual objects from camera
 rviz -d $(rospack find pr2eus_tutorials)/config/pr2_tabletop.rviz
 ```
 
+##### Run all programs by one launch file
+```bash
+# Using a real PR2 robot
+ssh <robot address>
+roslaunch pr2eus_tutorials pr2_tabletop_grasp.launch
+# Using a simulator on local machine
+roslaunch pr2eus_tutorials pr2_tabletop_grasp.launch gazebo:=true
+```
+
```

Comment:
`pr2_tabletop_grasp.launch` is deprecated.
in current implementation, we use `pr2_tabletop.launch` for the sample launch.
please merge use the launch, implement to use both rviz and image_view2 simaltaneously, and do not use `pr2_tabletop_grasp.launch`.

Reply:
i misunderstood.
please run both rviz and image_view2 simaltaneously and do not use argument `grasp_by_2d_point`.